### DISCUSSION TITLE
How to hardcode server URL in docker?

### DISCUSSION THREAD

**Discussion by Hydronicx** [jeffvli/feishin#634] [Q&A]
Hello, a bit of a noob question perhaps. I run Feishin in docker using: `docker run -d --name feishin -p 9180:9180 ghcr.io/jeffvli/feishin:latest`

I'm trying to hardcode the server URL (a navidrome server), but I can't figure out how to do it. I tried: `docker run -d --name feishin -p 9180:9180 ghcr.io/jeffvli/feishin:latest -e SERVER_NAME="my_server" -e SERVER_TYPE="navidrome" -e SERVER_URL="server_url" -e SERVER_LOCK=true` (with the actual server URL of course), but it doesn't work.

What am I doing wrong? Thanks.

**Comment by fanch317** [comment]
Hello,
Same problem here (im based on this [docker-compose.yml](https://github.com/jeffvli/feishin/issues/545#issuecomment-2053057097))

You can see that setting.js is ok :
![image](https://github.com/jeffvli/feishin/assets/1344319/56efbc93-de65-4340-bb2f-7710b4bb113b)

**Comment by fanch317** [comment]
It work but not like I expect.
This feature SERVER_LOCK is only for purpose to user to create a locked connection setting with a server.
The connection still has to be configured by the final user and this is confusing.
![image](https://github.com/jeffvli/feishin/assets/1344319/837d825f-4d0e-4916-957d-e1457c8a06d0)

A login page would have been appreciated rather than having to add a server.
I don't have the skills to help with this, but I will suggest clarifying it in the docs.
